lib/seahorn/BvOpSem2.cc

```diff
@@ -500,7 +500,9 @@ class OpSemVisitor : public InstVisitor<OpSemVisitor>, OpSemVisitorBase {
                              f->getName().startswith("verifier.nondet") ||
                              f->getName().startswith("__VERIFIER_nondet")))
         visitNondetCall(CS);
-      else
+      else if (f->getName().startswith("__sea_")) {
```

Comment:
This is not specific enough prefix. There might be many functions that we introduce with `__sea_` prefix.

Reply:
looks like this is still a todo

Reply:
I don't think this logic precludes adding new __sea_functions. We will need to add a sibling to visitFatPtrInstr(...) for each function. Overall, I remember we wanted to rename functions anyway. My preference is to make it
`__sea_extptr_[get|set|copy|recover] `
once merged with Xiang's implementation. This will also make the if condition tighter.

Reply:
I mean that you currently treat all functions with prefix `__sea_` as fat ptr related. But they don't have to be. `__sea_exptr` is a better, more specific prefix.

Reply:
Ok will make the change after rebase with HEAD.